sycl/test/on-device/basic_tests/kernel_bundle/kernel_bundle_api.cpp

```diff
@@ -13,6 +13,11 @@ class Kernel2Name;
 
 int main() {
   sycl::queue Q;
+
```

Comment:
can we add RUN lines per target device?

Reply:
Yes, we can. But it doesn't make a lot of sense to run the test on many devices many devices. We could specify only one, but it seems it is unknown/undefined which devices are available on CI machines.